docs-source/spring/content/starters/ucp.md

```diff
@@ -0,0 +1,104 @@
+---
+title: "Oracle Spring Boot Starter for Universal Connection Pool"
+---
+
+This starter provides a connection (data source) to an Oracle Database using Universal Connection Pool, which provides an efficient way
+to use database connections. 
+
+To add this starter to your project, add this Maven dependency:
+
+```xml
+<dependency>
+    <groupId>com.oracle.database.spring</groupId>
+    <artifactId>oracle-spring-boot-starter-ucp</artifactId>
+    <version>23.4.0</version>
+    <type>pom</type>
+</dependency>
+```
+
+For Gradle projects, add this dependency:
+
+```
+implementation 'com.oracle.database.spring:oracle-spring-boot-starter-ucp:23.4.0:pom'
+```
+
+An Oracle data source will be injected into your application and can be used normally. You must configure
+it as shown below, and you should also add either Spring Data JDBC or Spring Data JPA to your project.
```

Comment:
What is "it" referring to? The data source? The application?

"... as shown in the following example, and ..."